src/support/import-supervisor.js

```diff
@@ -10,9 +10,25 @@
  * governing permissions and limitations under the License.
  */
 
+import { IMPORT_JOB_STATUS } from '@adobe/spacecat-shared-data-access/src/models/importer/import-job.js';
```

Comment:
this could probably be exported as an interface in `index.t.js` in order to alleviate having to import a deep file location:
```js
export interface ImportJobStatus {
 a: 'a',
 ...
 }
```